fe2-android/app/src/main/java/com/github/dedis/popstellar/utility/handler/data/LinkedOrganizationsHandler.kt

```diff
@@ -76,15 +77,15 @@ constructor(
     // LAO. This might be changed in the future (making a pop-up asking the user if he/she wants
     // to subscribe to that)
     tokenExchange.tokens.forEach { t ->
-      laoRepo.addDisposable(
+      disposables.add(
```

Comment:
Using the disposable from the handler doesn't give you the opportunity to clear the added disposables when closing a lao or the application. Take a look at the usage of disposables in the RollCallRepository or any other repository.

Reply:
Oh ok, then should I create the disposables in LinkedOrganizationsRepository (as done in RollCallRepository) and then call it from the LinkedOrganizationsHandler ? Or should I do it directly in the handler ?

Reply:
Yep you can do it from the repository as for roll calls and keep it consistent